src/TaskManager/Plug-ins/Argo/ArgoPlugin.cs

```diff
@@ -635,7 +635,7 @@ private Messaging.Common.Storage GenerateIntermediaryArtifactStore(string artifa
             }
 
             var storageInfo = Event.IntermediateStorage.Clone() as Messaging.Common.Storage;
-            storageInfo!.RelativeRootPath = $"{storageInfo.RelativeRootPath}/{{{{ workflow.name }}}}/{artifactName}";
+            storageInfo!.RelativeRootPath = $"{storageInfo.RelativeRootPath}/{artifactName}";
```

Comment:
Just curious why remove the workflow name here

Reply:
This is due to us resolving the output artifacts before the argo task has been created, and the workflow name is only assigned when it has been created in argo, making it more complicated to include it in the path.

Reply:
Marked as resolved as we need this merged but if there's any issues feel free to message/raise them in standup on Tuesday